tools/python/influx-migration/influx_migration.py

```diff
@@ -49,6 +50,12 @@
 # mount an S3 bucket
 MOUNT_POINT_NAME = "influxdb-backups"
 
+# The recommended amount of time to wait before scraping from the /metrics endpoint
+# to ensure metrics are up to date
+METRICS_SCRAPE_INTERVAL_SECONDS=10
```

Comment:
10 seconds comes from [this example](https://docs.influxdata.com/flux/v0/prometheus/scrape-prometheus/#write-prometheus-metrics-to-influxdb-at-regular-intervals), as there is no official documentation of how long it may take for `/metrics` to update.

Reply:
Can we update the comment, as this this isn't the recommended amount of time, but just the scrape interval.

Reply:
Updated.